hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -261,19 +258,6 @@ public HoodieTableMetadata getMetadata() {
    */
   public abstract HoodieWriteMetadata<O> insertOverwriteTable(HoodieEngineContext context, String instantTime, I records);
 
-  /**
-   * Updates Metadata Indexes (like Column Stats index)
-   * TODO rebase onto metadata table (post RFC-27)
-   *
-   * @param context instance of {@link HoodieEngineContext}
-   * @param instantTime instant of the carried operation triggering the update
-   */
-  public abstract void updateMetadataIndexes(
```

Comment:
may I know how is this change is related to this patch ?

Reply:
So some of the tests that we have in "spark-client" module actually require `SparkAdapter` to be loaded, which lives in "hudi-spark" module, entailing that it couldn't be loaded there.

So i had to either move the tests to "hudi-spark" or remove this method (which uses `AvroConversionUtil`, in turn referencing `SparkAdapter`) which i'm removing regardless in another PR.